Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-ExchangeInformation.ps1

```diff
@@ -420,6 +421,12 @@ Function Get-ExchangeInformation {
             Write-Verbose "MAPI HTTP Enabled and Download Domains Enabled results not accurate"
         }
 
+        if ($OSMajorVersion -ge [HealthChecker.OSServerVersion]::Windows2016) {
```

Comment:
Maybe we should add a check for Edge here as well?

Reply:
Yeah, good point. Idea was to filter only on OS version and to do the rest of the Exchange role / CU logic within the `Get-AMSIConfigurationState` function. But good point since I'm not sure if the `Get-SettingOverride` cmdlet is available on Edge Transport or not which may cause FP if it's not available.

Reply:
Done. Added check for Edge transport server role. Will keep the check for Edge transport server role within the `Invoke-AnalyzerSecurityAMSIConfigState` function as well.